File: Python3/Length of Last Word.py
# 40ms, 100.0%
class Solution:
    def lengthOfLastWord(self, s):
        s = s.strip()
        if not s: return 0
        i, slen = -1, -len(s)
        while i > slen:
            if s[i - 1] == ' ': break;
            i -= 1
        return -i
        

File: Java/Length of Last Word.java
// 0ms, 100.00%; 37.4MB, 18.18%
class Solution {
    public int lengthOfLastWord(String s) {
        int res = 0;
        for (int i = s.length() - 1; i > -1; --i) {
            if (s.charAt(i) == ' ') {
                if (res > 0) {
                    break;
                }
            } else {
                res += 1;
            }
        }
        return res;
    }
}
